kolibri/core/tasks/validation.py

```diff
@@ -1,5 +1,44 @@
 from rest_framework import serializers
 
+from kolibri.core.serializers import DateTimeTzField
+
+
+class EnqueueArgsSerializer(serializers.Serializer):
+    """
+    A serializer for `enqueue_args` object of incoming user request data.
+    """
+
+    enqueue_at = DateTimeTzField(required=False)
+    enqueue_in = serializers.DurationField(required=False)
```

Comment:
Just realizing that we will need to have a utility function on the frontend to produce this duration value in the form `'[DD] [HH:[MM:]]ss[.uuuuuu]'`.